ykrt/src/mt.rs

```diff
@@ -70,7 +70,7 @@ pub struct MT {
     /// How many worker threads are currently running. Note that this may temporarily be `>`
     /// [`max_worker_threads`].
     active_worker_threads: AtomicUsize,
-    tracer: Arc<dyn Tracer>,
+    tracer: Arc<Arc<dyn Tracer>>,
```

Comment:
Go on then: why the double `Arc`?

Reply:
Good question. Will check.

Reply:
It took me a while to remind myself what I was doing here: this was a midway point on a change to allow the `Tracer` to be changed *during* a yk run (at the moment this isn't useful, but one day it might be). But `Arc<Arc>` is pointless in and of itself -- I just forgot to complete things! d4ee063 completes the change.

Reply:
You could have said that you were doing something very very clever, and I'd have believed you :)

Reply:
I admire your faith, but it is always more likely that I'm doing something very very stupid :p